[Dashboard][Processor] Add option to disable a creation of default http trigger (#3051)
nuclio/nuclio: hack/examples/README.md

To help you make the most of Nuclio, the following function examples are provided:

> Note: all functions contain field `disableDefaultHttpTrigger: false` to stay deployable even if default http trigger creation is disabled on platform configuration level. 

Reviewer: ```suggestion
Note: all function examples have the explicit field `disableDefaultHttpTrigger: false`, so they are deployable even if default http trigger creation is disabled on platform configuration level. 
```

> Note: all function examples have the explicit field `disableDefaultHttpTrigger: false`, so they are deployable even if default http trigger creation is disabled on platform configuration level.

Reviewer 2: need to add reference to `docs/reference/function-configuration/function-configuration-reference.md`

Reviewer 2: @rokatyy - Perhaps I was not clear enough.
What I meant is that you should add information as well on `docs/reference/function-configuration/function-configuration-reference.md` to make sure this file explains the new configuration as well.

Author: @liranbg added :)
